cargo-concordium/src/build.rs

```diff
@@ -419,6 +422,46 @@ fn write_schema_json(
     Ok(())
 }
 
+/// Write the provided schema in its base64 representation to a file inside the
+/// `root` directory if `Some(root)` is given.
+/// Print the provided schema in its base64 representation to the console if
+/// `schema_base64_log` is true.
+pub fn write_schema_base64(
+    root: Option<PathBuf>,
+    schema: &VersionedModuleSchema,
+    schema_base64_log: bool,
+) -> anyhow::Result<()> {
+    let schema_base64 = ENCODER.encode(to_bytes(schema));
+
+    // printing base64 schema to console
+    if schema_base64_log {
+        println!(
+            "   The base64 conversion of the schema is:\n{}",
+            schema_base64
+        );
+    }
+
+    // writing base64 schema to file
+    if let Some(root) = root {
+        let mut out_path = root;
+
+        let manifest = Manifest::from_path("Cargo.toml").context("Could not read Cargo.toml.")?;
+
+        let package = manifest
+            .package
+            .context("Manifest needs to specify [package]")?;
+
+        out_path.push(to_snake_case(package.name) + "_schema.b64");
```

Comment:
I think the logic here should be the same as for `--schema`, namely that they provide a **file name**. Since only one file will be written.